test/integration/features/cmd-config.feature

```diff
@@ -0,0 +1,242 @@
+@cmd-config @command
+Feature: Minishift config subcommands
+Commands `minishift config [sub-command]` are used for storing
+user defined options which changes default behaviour of Minishift.
+
+  @minishift-only
+  Scenario: Config is empty
+    Given Minishift has state "Does Not Exist"
+     Then executing "minishift config view" succeeds
+      And stdout should be empty
+
+  Scenario Outline: Setting values in bad range or format
+  Minishift config set should not set a value, if this value is of bad format or in bad range.
+     When executing "minishift config set <property> <value>" fails
+     Then JSON config file "config/config.json" does not contain key "<property>" with value matching "<value>"
+      And stdout of command "minishift config get <property>" is equal to "<nil>"
+
+  Examples: Wrong values for unit based keys
+    | property        | value     |
+    | disk-size       | 0         |
+    | disk-size       | 25000.78  |
+    | disk-size       | much more |
+    | disk-size       | [2g 5g]   |
```

Comment:
interesting ;-)... but how about minus values? In code we actually check for this:
https://github.com/minishift/minishift/blob/d81c3d2a53e07c73234972e97c4c712738f2a599/pkg/minishift/config/validations.go#L73